sdk/schemaregistry/azure-data-schemaregistry/src/main/java/com/azure/data/schemaregistry/implementation/AzureSchemaRegistry.java

```diff
@@ -80,7 +80,7 @@ public Schemas getSchemas() {
                         .policies(new UserAgentPolicy(), new RetryPolicy(), new CookiePolicy())
                         .build(),
                 JacksonAdapter.createDefaultSerializerAdapter(),
-                endpoint);
+                endpoint, "2020-09-01-preview");
```

Comment:
nit: should it be  `SchemaRegistryVersion.getLatest()` or `SchemaRegistryVersion.V2020_09_01_PREVIEW` instead of literal